pages/linux/xar.md

```diff
@@ -0,0 +1,15 @@
+# xar
+
+> Manage .xar archives.
+
+- Create a xar archive of all files in {{directory}}:
+
+`xar -cf sample.xar {{directory}}`
```

Comment:
Perhaps this would be better as `{{path/to/directory}}` with a description something like ....`in the specified directory:`? Then it would be more consistent with the existing pages we've got already.